### PR TITLE
Voron2.4 cable organizers; Crydom SSR terminal guard

### PR DESCRIPTION
**Pull request by evandepol**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [ ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

- Safety guard for Crydom GN series SSR terminals
- set of cable organizing arms for Voron 2.4
 
#### How was it tested? 

Printed and installed on my own build. Pictures of the installed components are included.

#### Any background context you want to provide?

Included in Readme.MD files.

#### Screenshots (if appropriate)

Included in the PR.

### CODE REVIEW

File: printer_mods/evandepol/Crydom-GN-Guard/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Crydom GN series terminal guard
+
+I replaced the SSR on my build with a Crydom GN series SSR. The model i received didn't have the terminal cover, so I create a simple cover that is secured with the mounting screw on the SSR, and helps prevent accidental touching of the high voltage terminals.
```
Comment (KiloQubit) [inline printer_mods/evandepol/Crydom-GN-Guard/README.md:3]: Replace `I create` with `I created`.

File: printer_mods/evandepol/Crydom-GN-Guard/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+![Model](Images/guard.png)
+
```
Comment (KiloQubit) [inline printer_mods/evandepol/Crydom-GN-Guard/README.md:6]: The mounting hole on top prints in air, it needs a chamfer or some other built-in way to support it.

Screen Shot 2021-05-19 at 08 06 09

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+An example showing the wiring on my Voron build:
+
+![Dressed wiring](Images/dressed.jpg)
+
+
+## Printing
+
+Parts are intended to be printed in the provided orientation (on the side). I Used ABS, standard Voron print settings.
+
```
Comment (KiloQubit) [inline printer_mods/evandepol/Voron2.4-Cable-clamps/README.md:36]: Please add your `Images/detail.jpg` to the README, or delete that image file.

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Cable clamps
+
+I used this set of clamps to route and secure the wiring in my Voron 2.4 350mm.
+
+There are three three types, each requiring an m3 bolt and t-nut in order to secure them to the inside perimeter of the upper (meaning closest to the deck) 2020 rail. The deck should rest on top of the clamps.
```
Comment (KiloQubit) [inline printer_mods/evandepol/Voron2.4-Cable-clamps/README.md:5]: Replace `three three types` with `three types`.

Replace `2020 rail` with `2020 extrusion`.

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+openscad files are provided for your own tinkering needs.
+
+
+## Clamp types
+
+The first type only secures along the perimeter. I used this to secure high voltage wiring from the power inlet to the power supplies. Multi "tiers" of zip ties can be used to keep different types of wiring separated.
+
+![Clamp](Images/edge-clamp.png)
+
+The second type provides a horizontal bar that secures to the inside to the 2020 rail, and can be used to lash wiring to and keep it secure and organized. I used one of those to guide the high voltage wiring.
```
Comment (KiloQubit) [inline printer_mods/evandepol/Voron2.4-Cable-clamps/README.md:20]: Replace `2020 rail` with `2020 extrusion`.

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+![Horizontal bar](Images/cable-mount.png)
+
+The third type combines these two, and is used to route wiring along the perimeter, as well as guide nd secure wiring to various devices. I used several of these to provide wire tracks towards the Raspberry Pi, SKR boards, and SSR.
```
Comment (KiloQubit) [inline printer_mods/evandepol/Voron2.4-Cable-clamps/README.md:24]: Replace `guide nd secure` with `guide and secure`

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+![Combination](Images/combination.png)
+
```
Comment (KiloQubit) [inline printer_mods/evandepol/Voron2.4-Cable-clamps/README.md:27]: Both `Combination-Clamp-and-Ramp-350mm.stl` and `DIN-Rail-Cable-Mount-350mm.stl` have too steep of an overhang at the top of the mounting hole, please ensure overhangs are no steeper than 45 degrees.

Screen Shot 2021-05-19 at 08 22 47

Screen Shot 2021-05-19 at 08 23 24

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for you mod!  Please see requested changes below.

File: printer_mods/evandepol/Crydom-GN-Guard/README.md (hunk 1 above)
Comment (evandepol) [reply printer_mods/evandepol/Crydom-GN-Guard/README.md]: Fixed.

File: printer_mods/evandepol/Crydom-GN-Guard/README.md (hunk 2 above)
Comment (evandepol) [reply printer_mods/evandepol/Crydom-GN-Guard/README.md:6]: Fixed by using a bridging approach over 2 slicing layers. Performing a test print now to confirm but should be clean.

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md (hunk 3 above)
Comment (evandepol) [reply printer_mods/evandepol/Voron2.4-Cable-clamps/README.md:40]: Added

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md (hunk 4 above)
Comment (evandepol) [reply printer_mods/evandepol/Voron2.4-Cable-clamps/README.md]: Fixed.

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md (hunk 5 above)
Comment (evandepol) [reply printer_mods/evandepol/Voron2.4-Cable-clamps/README.md]: Fixed.

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md (hunk 6 above)
Comment (evandepol) [reply printer_mods/evandepol/Voron2.4-Cable-clamps/README.md]: Fixed.

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md (hunk 7 above)
Comment (evandepol) [reply printer_mods/evandepol/Voron2.4-Cable-clamps/README.md:27]: Working on this. The next commit on this branch should address this. Will post back when it's ready.

File: printer_mods/evandepol/Crydom-GN-Guard/README.md (hunk 2 above)
Comment (evandepol) [reply printer_mods/evandepol/Crydom-GN-Guard/README.md:6]: IMG_0898

File: printer_mods/evandepol/Voron2.4-Cable-clamps/README.md (hunk 7 above)
Comment (evandepol) [reply printer_mods/evandepol/Voron2.4-Cable-clamps/README.md:27]: Updated and simplified; this now uses bridging where the screw access hole is.

**Comment by evandepol** [comment]
Hi @KiloQubit,thanks for the thorough review. I made the suggested edits, let me know if you have additional feedback.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**